stable/fairwinds-insights/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.7.12
```

Comment:
This is a breaking change--we should change to 0.8.0

Reply:
done